### ISSUE TITLE
Bug while importing books from my local library

### ISSUE THREAD

**Issue by charlotteburki-conser** [Leseratte10/acsm-calibre-plugin#25]
### Bug description

Importing books from Adobe's test library works well. But I have an error when trying to import books from a local library, any help would be appreciated.

```
[1.72] Started up in 1.72 seconds with 17 books
Starting QuickView
Running file type plugin DeACSM failed with traceback:
Traceback (most recent call last):
  File "calibre/customize/ui.py", line 175, in _run_filetype_plugins
  File "calibre_plugins.deacsm.__init__", line 378, in run
    success, replyData = fulfill(path_to_ebook, deacsmprefs["notify_fulfillment"])
  File "calibre_plugins.deacsm.libadobeFulfill", line 439, in fulfill
    updateLoanReturnData(adobe_fulfill_response)
  File "calibre_plugins.deacsm.libadobeFulfill", line 479, in updateLoanReturnData
    deviceUUID = fulfillmentResultToken.find("./%s/%s/%s/%s" % (adNS("fulfillmentResult"), adNS("resourceItemInfo"), adNS("licenseToken"), adNS("device"))).text
AttributeError: 'NoneType' object has no attribute 'text'
```

Thanks a lot for this awesome plugging! 

### Operating system

MacOS

### Which version of Calibre are you running?

5.42.0

### Which version of the DeACSM plugin are you running?

v0.0.15

**Comment by Leseratte10** [comment]
Thanks for the report. This bug should already be fixed by dd104665bb27a87f941243746cce4a0db7a24e79, though I have not yet released a new version since then. 

You can download the latest master with the bugfixes either from the Github actions page (go to https://github.com/Leseratte10/acsm-calibre-plugin/actions , choose newest entry, scroll down to "Artifacts" then click "linux" (even though you're on Mac) ) or from the 2nd post of the Mobileread thread ( https://www.mobileread.com/forums/showpost.php?p=4157570&postcount=2 ) which also contains the newest version.

If you still get this error message with that version of the plugin please post a new log.

**Comment by charlotteburki-conser** [comment]
Yes, all working well with the beta version, thanks.
